frontend/src/pages/_app.tsx

```diff
@@ -13,20 +14,42 @@ import 'styles/global.scss'
 
 function CustomApp(props: AppProps) {
   const { Component, pageProps } = props
+  const router = useRouter()
   const { i18n } = useTranslation()
+  const { initialize, trackEvent } = useGTM()
 
   useEffect(() => {
     // Remove the server-side injected CSS.
     const jssStyles = document.querySelector('#jss-server-side')
     if (jssStyles && jssStyles.parentElement) {
       jssStyles.parentElement.removeChild(jssStyles)
     }
+  }, [])
 
-    // Init GTM
-    useGTM().initialize({
+  useEffect(() => {
```

Comment:
Good call to use dedicated effect for the gtm :rocket: 

As I see we're going to `initialize` gtm on every language change now.

I can't evaulate if this is okay or not, do you have a view on that?

If it's not okay, we can use a 3rd effect just for the init method with no dependencies

Reply:
I decided to put GTM initialization back to the first useEffect after all. It is better to call it only once.